### PR TITLE
Add why section in README

### PR DESCRIPTION
**Pull request by instabledesign**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,6 +10,11 @@
 
 Gol4ng/Logger is another GO logger. The main line is to provide a friendly and fast API to send your log whenever you want. 
 
+## Why another one?
+
+When i start go i search a logger that can be **simple to use**, **efficient**, **multi output**, **multi formats** and quite easy to **extend**.
```
Comment (rmasclef) [inline README.md]: When I started Go I searched a logger that can be **simple to use**, **efficient**, **multi output**, **multi formats** and quite easy to **extend**.

File: README.md [hunk 2]
```diff
@@ -10,6 +10,11 @@
 
 Gol4ng/Logger is another GO logger. The main line is to provide a friendly and fast API to send your log whenever you want. 
 
+## Why another one?
+
+When i start go i search a logger that can be **simple to use**, **efficient**, **multi output**, **multi formats** and quite easy to **extend**. 
+That's why i create this logger with built in [handlers](#Handlers)(process a log), [formatters](#formatters)(format log in another representation), [middlewares](#middlewares)(log modification before handler)
```
Comment (rmasclef) [inline README.md]: That's why I created this logger with built-in [handlers](#Handlers)(process a log), [formatters](#formatters)(format log in another representation), [middlewares](#middlewares)(log modification before handler)

File: README.md [hunk 3]
```diff
@@ -111,11 +116,10 @@ Available writer:
 - **compress** _it will compress log to gzip/zlib_
 - **gelf_chunked** _it will chunked log entry into gelf chunk_
 - **rotate** _it will write in io.Writer and rotate writer on demand_
-- **time rotate** _it's a rotate writer that rorate with `time.Ticker`_
+- **time_rotate** _it's a rotate writer that rorate with `time.Ticker`_
```
Comment (rmasclef) [inline README.md]: rorate -> rotates

Comment (rmasclef) [reply README.md]: ?

Comment (instabledesign) [reply README.md]: now i see the `rorate` i think the **s** was missing but in fact it's the ro**T**ate

Comment (instabledesign) [reply README.md]: fixed directly on master by 3cb5b11b474b0a1a3fde0eeeb7682e7d1a739158

**Pull request merged**